testing/utilities.py

```diff
@@ -371,7 +371,7 @@ def results_to_df(self, points, start_time, final_time, url='http://127.0.0.1:50
 
         df = pd.DataFrame()
         for point in points:
-            res = requests.put('{0}/results'.format(url), data={'point_name':point,'start_time':start_time, 'final_time':final_time}).json()['payload']
+            res = requests.put('{0}/results'.format(url), data={'point_names':[point],'start_time':start_time, 'final_time':final_time}).json()['payload']
             df = pd.concat((df,pd.DataFrame(data=res[point], index=res['time'],columns=[point])), axis=1)
```

Comment:
I think the for-loop and concatenation can be removed since now the points are provided in a list all at once.

Reply:
I updated the pull request removing the for cycle and also found and corrected a small bug.